src/theme/variables.css

```diff
@@ -396,4 +397,28 @@ hr {
   .tablet {
     display: unset;
   }
+
+  .header {
+    display: grid;
+    grid: "id       timeline" min-content
+          "cards    timeline" 1fr
+          / 1fr 500px;
+    justify-content: space-between;  
+  }
+
+  .id {
+    grid-area: id;
+  }
+  
+  .cards {
```

Comment:
cards is not a good class name, this is "info"

Reply:
Ok sir